x-pack/platform/test/plugin_api_integration/test_suites/task_manager/check_registered_task_types.ts

```diff
@@ -213,6 +212,7 @@ export default function ({ getService }: FtrProviderContext) {
         'security:telemetry-timelines',
         'session_cleanup',
         'slo:bulk-delete-task',
+        'slo:orphan-summary-cleanup-task',
```

Comment:
@kdelemme Task manager uses the exact key to get task definition. If you change from UPPER to lower (or vice versa), any existing tasks persisted with the old type will no longer match a registered definition. They won’t be claimed/run and will remain in the index. New scheduling using the new key will create a separate task type. 
Task definition and abortController usage LGTM, but I think we should keep the same task type